src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
+
+    multiply(a, b) {
+      return a * b;
+    },
+
+    divide(a, b) {
+      return a / b;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, num) {
+      this.result = operation(this.result, num);
```

Comment:
[CODE KNOWLEDGE] - Since `operate` is a method in the object, you don't need to use the `function` keyword, use the method shortcut instead. Change `operate(operation, num) {` to `operate(operation, num) {`.